Resources/Prototypes/_NF/Catalog/Fills/Crates/nfsdarmory.yml

```
@@ -93,21 +93,21 @@
     - id: WeaponRevolverInspector
       amount: 4
 
-# - type: entity # Frontier - TODO: Restore this later
-  # id: CrateArmoryNonLethalNfsd
-  # parent: CrateNfsdSecure1
-  # name: nfsd non-lethal armory crate
-  # description: Contains a standard armory stock kit.
-  # components:
-  # - type: StorageFill
-    # contents:
-    # - id: WeaponDisabler
-      # amount: 5
-    # - id: WeaponPistolMk58Nonlethal
-      # amount: 4
-    # - id: WeaponDisablerSMG
-      # amount: 5
-    # - id: WeaponRifleLecterRubber
-      # amount: 4
-    # - id: WeaponShotgunEnforcerRubber
-      # amount: 4
+- type: entity
+  id: CrateArmoryNonLethalNfsd
+  parent: CrateNfsdSecure1
+  name: nfsd non-lethal armory crate
+  description: Contains a standard armory stock kit.
```

Non-lethal in description?
```suggestion
  description: Contains a non-lethal armory stock kit.
```